src/components/Tree/Tree.tsx

```diff
@@ -118,20 +116,22 @@ export const Tree: FC<TreeProps> = ({
     }, []);
 
     return (
-        <DndProvider backend={HTML5Backend}>
-            <ul
-                data-test-id="tree"
-                className="tw-p-0 tw-m-0 tw-font-sans tw-font-normal tw-list-none tw-text-left tw-text-sm tw-select-none"
-            >
-                {renderNodeArray({
-                    nodes: treeNodes,
-                    treeName,
-                    activeIds,
-                    onClick: onNodeClick,
-                    onDrop: handleDrop,
-                    onEditableSave: handleEditableSave,
-                })}
-            </ul>
-        </DndProvider>
+        <div id={treeName}>
```

Comment:
Why is treeName used as id? Kind of misleading. Should it be treeId or just id?

Reply:
Renamed it